cmd/podman/save.go

```diff
@@ -128,12 +128,15 @@ func saveCmd(c *cli.Context) error {
 	// need dest to be in the format transport:path:reference for the following transports
 	if (strings.Contains(dst, libpod.OCIArchive) || strings.Contains(dst, libpod.DockerArchive)) && !strings.Contains(newImage.ID(), source) {
 		prepend := ""
-		if !strings.Contains(source, libpodImage.DefaultLocalRepo) {
+		localhostReg := fmt.Sprintf("%s/", libpodImage.DefaultLocalRegistry)
+		if !strings.HasPrefix(source, libpodImage.DefaultLocalRegistry) {
```

Comment:
This should use `localhostReg` as well.

(Non-blocking BTW `localhostReg` hardcodes knowledge about the contents of the `DefaultLocalRegistry` constant, which it ideally shouldn’t do. `defaultRegistryPrefix` maybe, or just `prefix` given the very small scope?)